packages/extension-browser/package.json

```diff
@@ -6,7 +6,7 @@
     ],
     "timeout": "1m"
   },
-  "bundleSize": 700000,
+  "bundleSize": 650000,
```

Comment:
Reducing it even more 👏👏👏

Reply:
Yeah, given the recent savings I figured it was time to lock-in the lower threshold. 😉